.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,216 @@
+      - name: Set up JDK 21
+        uses: actions/setup-java@v3
+        with:
+          java-version: '21'
+          distribution: 'temurin'
+      
+      - name: Make all gradlew files executable
+        run: |
+          find project/backend -name "gradlew" -type f -exec chmod +x {} \;
+      
+      - name: Cache Gradle packages
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)